### DISCUSSION TITLE
关于tabs的使用问题 a question about the use of el-tabs

### DISCUSSION THREAD

**Discussion by D-MHao** [element-plus/element-plus#18609] [Q&A] [answered]
我在代码中使用了el-tabs 但是无论我怎么切换tab-pane都不会切换以下是我的代码：
el-tabs don't work while i click on tab-pane,it don't change. Here is the code:

```
<template>
  <el-container>
    <el-header>
      质押融资管理系统
    </el-header>
    <el-container style="height: 100vh;">
      <el-tabs v-model="activeName" tab-position="left"  @tab-change="tabChange">
        <el-tab-pane label="质押信息" name="Pledge"><Pledge/></el-tab-pane>
        <el-tab-pane label="银行信息" name="Bank">Bank</el-tab-pane>
        <el-tab-pane label="从excel中导入" name="Import">import</el-tab-pane>
        <el-tab-pane label="导出为excel" name="Export">export</el-tab-pane>
      </el-tabs>
    </el-container>
  </el-container>
</template>

<script>
import Pledge from './components/Pledge.vue';

export default {
  name: 'App',
  components: {
    Pledge
  },
  data(){
    return{
      activeName:"Pledge",
    }
  },
  methods:{
    tabChange(){
      console.log("tabchange activeName:"+this.activeName)
    }
  }
  
}
</script>
```

![image](https://github.com/user-attachments/assets/a829c220-f960-479c-84ab-5a51d40e13f2)

**Comment by warmthsea** [comment]
看起来是正常的 [Playground Demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtY29udGFpbmVyPlxuICAgIDxlbC1oZWFkZXI+XG4gICAgICDotKjmirzono3otYTnrqHnkIbns7vnu58te3thY3RpdmVOYW1lfX1cbiAgICA8L2VsLWhlYWRlcj5cbiAgICA8ZWwtY29udGFpbmVyIHN0eWxlPVwiaGVpZ2h0OiAxMDB2aDtcIj5cbiAgICAgIDxlbC10YWJzIHYtbW9kZWw9XCJhY3RpdmVOYW1lXCIgdGFiLXBvc2l0aW9uPVwibGVmdFwiICBAdGFiLWNoYW5nZT1cInRhYkNoYW5nZVwiPlxuICAgICAgICA8ZWwtdGFiLXBhbmUgbGFiZWw9XCLotKjmirzkv6Hmga9cIiBuYW1lPVwiUGxlZGdlXCI+PFBsZWRnZS8+PC9lbC10YWItcGFuZT5cbiAgICAgICAgPGVsLXRhYi1wYW5lIGxhYmVsPVwi6ZO26KGM5L+h5oGvXCIgbmFtZT1cIkJhbmtcIj5CYW5rPC9lbC10YWItcGFuZT5cbiAgICAgICAgPGVsLXRhYi1wYW5lIGxhYmVsPVwi5LuOZXhjZWzkuK3lr7zlhaVcIiBuYW1lPVwiSW1wb3J0XCI+aW1wb3J0PC9lbC10YWItcGFuZT5cbiAgICAgICAgPGVsLXRhYi1wYW5lIGxhYmVsPVwi5a+85Ye65Li6ZXhjZWxcIiBuYW1lPVwiRXhwb3J0XCI+ZXhwb3J0PC9lbC10YWItcGFuZT5cbiAgICAgIDwvZWwtdGFicz5cbiAgICA8L2VsLWNvbnRhaW5lcj5cbiAgPC9lbC1jb250YWluZXI+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0PlxuaW1wb3J0IFBsZWRnZSBmcm9tICcuL1BsZWRnZS52dWUnO1xuXG5leHBvcnQgZGVmYXVsdCB7XG4gIG5hbWU6ICdBcHAnLFxuICBjb21wb25lbnRzOiB7XG4gICAgUGxlZGdlXG4gIH0sXG4gIGRhdGEoKXtcbiAgICByZXR1cm57XG4gICAgICBhY3RpdmVOYW1lOlwiUGxlZGdlXCIsXG4gICAgfVxuICB9LFxuICBtZXRob2RzOntcbiAgICB0YWJDaGFuZ2UoKXtcbiAgICAgIGNvbnNvbGUubG9nKFwidGFiY2hhbmdlIGFjdGl2ZU5hbWU6XCIrdGhpcy5hY3RpdmVOYW1lKVxuICAgIH1cbiAgfVxuICBcbn1cbjwvc2NyaXB0PiIsImVsZW1lbnQtcGx1cy5qcyI6ImltcG9ydCBFbGVtZW50UGx1cyBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5cbmxldCBpbnN0YWxsZWQgPSBmYWxzZVxuYXdhaXQgbG9hZFN0eWxlKClcblxuZXhwb3J0IGZ1bmN0aW9uIHNldHVwRWxlbWVudFBsdXMoKSB7XG4gIGlmIChpbnN0YWxsZWQpIHJldHVyblxuICBjb25zdCBpbnN0YW5jZSA9IGdldEN1cnJlbnRJbnN0YW5jZSgpXG4gIGluc3RhbmNlLmFwcENvbnRleHQuYXBwLnVzZShFbGVtZW50UGx1cylcbiAgaW5zdGFsbGVkID0gdHJ1ZVxufVxuXG5leHBvcnQgZnVuY3Rpb24gbG9hZFN0eWxlKCkge1xuICBjb25zdCBzdHlsZXMgPSBbJ2h0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vZWxlbWVudC1wbHVzQGxhdGVzdC9kaXN0L2luZGV4LmNzcycsICdodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL2VsZW1lbnQtcGx1c0BsYXRlc3QvdGhlbWUtY2hhbGsvZGFyay9jc3MtdmFycy5jc3MnXS5tYXAoKHN0eWxlKSA9PiB7XG4gICAgcmV0dXJuIG5ldyBQcm9taXNlKChyZXNvbHZlLCByZWplY3QpID0+IHtcbiAgICAgIGNvbnN0IGxpbmsgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaW5rJylcbiAgICAgIGxpbmsucmVsID0gJ3N0eWxlc2hlZXQnXG4gICAgICBsaW5rLmhyZWYgPSBzdHlsZVxuICAgICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdsb2FkJywgcmVzb2x2ZSlcbiAgICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignZXJyb3InLCByZWplY3QpXG4gICAgICBkb2N1bWVudC5ib2R5LmFwcGVuZChsaW5rKVxuICAgIH0pXG4gIH0pXG4gIHJldHVybiBQcm9taXNlLmFsbFNldHRsZWQoc3R5bGVzKVxufVxuIiwidHNjb25maWcuanNvbiI6IntcbiAgXCJjb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IFwiRVNOZXh0XCIsXG4gICAgXCJqc3hcIjogXCJwcmVzZXJ2ZVwiLFxuICAgIFwibW9kdWxlXCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVSZXNvbHV0aW9uXCI6IFwiQnVuZGxlclwiLFxuICAgIFwidHlwZXNcIjogW1wiZWxlbWVudC1wbHVzL2dsb2JhbC5kLnRzXCJdLFxuICAgIFwiYWxsb3dJbXBvcnRpbmdUc0V4dGVuc2lvbnNcIjogdHJ1ZSxcbiAgICBcImFsbG93SnNcIjogdHJ1ZSxcbiAgICBcImNoZWNrSnNcIjogdHJ1ZVxuICB9LFxuICBcInZ1ZUNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogMy4zXG4gIH1cbn1cbiIsIlBsYXlncm91bmRNYWluLnZ1ZSI6IjxzY3JpcHQgc2V0dXA+XG5pbXBvcnQgQXBwIGZyb20gJy4vQXBwLnZ1ZSdcbmltcG9ydCB7IHNldHVwRWxlbWVudFBsdXMgfSBmcm9tICcuL2VsZW1lbnQtcGx1cy5qcydcbnNldHVwRWxlbWVudFBsdXMoKVxuPC9zY3JpcHQ+XG5cbjx0ZW1wbGF0ZT5cbiAgPEFwcCAvPlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcInZ1ZVwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9ydW50aW1lLWRvbUBsYXRlc3QvZGlzdC9ydW50aW1lLWRvbS5lc20tYnJvd3Nlci5qc1wiLFxuICAgIFwiQHZ1ZS9zaGFyZWRcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0B2dWUvc2hhcmVkQGxhdGVzdC9kaXN0L3NoYXJlZC5lc20tYnVuZGxlci5qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9lbGVtZW50LXBsdXNAbGF0ZXN0L2Rpc3QvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9lbGVtZW50LXBsdXNAbGF0ZXN0L1wiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsIlBsZWRnZS52dWUiOiI8dGVtcGxhdGU+XHJcbiAgPGRpdiBjbGFzcz1cIm1iLTRcIj5cclxuICAgIDxlbC1idXR0b24+RGVmYXVsdDwvZWwtYnV0dG9uPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiPlByaW1hcnk8L2VsLWJ1dHRvbj5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cInN1Y2Nlc3NcIj5TdWNjZXNzPC9lbC1idXR0b24+XHJcbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJpbmZvXCI+SW5mbzwvZWwtYnV0dG9uPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwid2FybmluZ1wiPldhcm5pbmc8L2VsLWJ1dHRvbj5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cImRhbmdlclwiPkRhbmdlcjwvZWwtYnV0dG9uPlxyXG4gIDwvZGl2PlxyXG5cclxuICA8ZGl2IGNsYXNzPVwibWItNFwiPlxyXG4gICAgPGVsLWJ1dHRvbiBwbGFpbj5QbGFpbjwvZWwtYnV0dG9uPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiIHBsYWluPlByaW1hcnk8L2VsLWJ1dHRvbj5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cInN1Y2Nlc3NcIiBwbGFpbj5TdWNjZXNzPC9lbC1idXR0b24+XHJcbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJpbmZvXCIgcGxhaW4+SW5mbzwvZWwtYnV0dG9uPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwid2FybmluZ1wiIHBsYWluPldhcm5pbmc8L2VsLWJ1dHRvbj5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cImRhbmdlclwiIHBsYWluPkRhbmdlcjwvZWwtYnV0dG9uPlxyXG4gIDwvZGl2PlxyXG5cclxuICA8ZGl2IGNsYXNzPVwibWItNFwiPlxyXG4gICAgPGVsLWJ1dHRvbiByb3VuZD5Sb3VuZDwvZWwtYnV0dG9uPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiIHJvdW5kPlByaW1hcnk8L2VsLWJ1dHRvbj5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cInN1Y2Nlc3NcIiByb3VuZD5TdWNjZXNzPC9lbC1idXR0b24+XHJcbiAgICA8ZWwtYnV0dG9uIHR5cGU9XCJpbmZvXCIgcm91bmQ+SW5mbzwvZWwtYnV0dG9uPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwid2FybmluZ1wiIHJvdW5kPldhcm5pbmc8L2VsLWJ1dHRvbj5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cImRhbmdlclwiIHJvdW5kPkRhbmdlcjwvZWwtYnV0dG9uPlxyXG4gIDwvZGl2PlxyXG5cclxuICA8ZGl2PlxyXG4gICAgPGVsLWJ1dHRvbiA6aWNvbj1cIlNlYXJjaFwiIGNpcmNsZSAvPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwicHJpbWFyeVwiIDppY29uPVwiRWRpdFwiIGNpcmNsZSAvPlxyXG4gICAgPGVsLWJ1dHRvbiB0eXBlPVwic3VjY2Vzc1wiIDppY29uPVwiQ2hlY2tcIiBjaXJjbGUgLz5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cImluZm9cIiA6aWNvbj1cIk1lc3NhZ2VcIiBjaXJjbGUgLz5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cIndhcm5pbmdcIiA6aWNvbj1cIlN0YXJcIiBjaXJjbGUgLz5cclxuICAgIDxlbC1idXR0b24gdHlwZT1cImRhbmdlclwiIDppY29uPVwiRGVsZXRlXCIgY2lyY2xlIC8+XHJcbiAgPC9kaXY+XHJcbjwvdGVtcGxhdGU+XHJcblxyXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxyXG5pbXBvcnQge1xyXG4gIENoZWNrLFxyXG4gIERlbGV0ZSxcclxuICBFZGl0LFxyXG4gIE1lc3NhZ2UsXHJcbiAgU2VhcmNoLFxyXG4gIFN0YXIsXHJcbn0gZnJvbSAnQGVsZW1lbnQtcGx1cy9pY29ucy12dWUnXHJcbjwvc2NyaXB0PlxyXG4iLCJfbyI6e319)

**Reply by D-MHao** [reply]
非常神奇我自己整的时候就不行，项目目前可以通过http://zyrzv4.mc-cmh.fun:25921/访问 不管怎么切换 都不行。我得怀疑自己是不是哪里配置错误了

**Comment by D-MHao** [comment, marked as answer]
解决了 虽然原因不是很明确 但是使用vue ui进行依赖管理并更新依赖之后 就解决了
